docs/user/upgrade.rst

```diff
@@ -90,6 +90,16 @@ Compatibility Matrix
 | Dependencies | N     | N     | Y     |
 +--------------+-------+-------+-------+
 
+Version 2.3.0 Upgrade Guide
++++++++++++++++++++++++++++
+
+Default Query Planning Type
+---------------------------
+
+GeoMesa 2.3.0 changes the default query planning type from stat-based to heuristic-based. This will only affect the
+Accumulo data store, as other stores have not implemented statistics. To enable stat-based query planning, refer
+to :ref:`query_planning_hint`.
```

Comment:
With this default... does GeoMesa on Accumulo generate the stats tables by default?  Or does one have to opt into that?

Reply:
it will generate it by default, but you can turn it off in the data store params or via sys property.

Reply:
so it will still track stats, just won't use them for query planning. But they are available for bounds, counts, etc